cmd/helm/status.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"github.com/deislabs/porter-helm/pkg/helm"
+	"github.com/spf13/cobra"
+)
+
+func buildStatusCommand(m *helm.Mixin) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "status",
+		Short: "Execute the status functionality of this mixin",
```

Comment:
Do we have a more targeted description that we can supply that is specific to helm? 

Maybe, `Print the status of the helm components in the bundle`?

Reply:
Good call; will update as recommended.